src/react/components/pages/editorPage/editorSideBar.tsx

```diff
@@ -56,6 +63,8 @@ export default class EditorSideBar extends React.Component<IEditorSideBarProps,
                             rowHeight={233}
                             rowRenderer={this.rowRenderer}
                             overscanRowCount={2}
+                            scrollToAlignment="center"
```

Comment:
align the selected asset to middle of the list

Reply:
What other options does it support?  Maybe "top" would be better if that is an option?

Reply:
It also support "start" and "end".  Honestly, I prefer center since I tend to focus on the center.

https://github.com/bvaughn/react-virtualized/blob/master/docs/List.md

Reply:
Agree probably "center" is best option in our scenario.

Generally speaking the scrolling after a GOTO on listview is "start" or "end" depending on where you come from (previous selection).  If you want to easily try this open powerpoint or preview on macOS and try find or goto different page on preview.